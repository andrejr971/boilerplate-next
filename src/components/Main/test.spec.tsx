import { screen } from '@testing-library/react';
import { renderWithTheme } from 'utils/tests/renderWithTheme';
import { Main } from '.';

describe('<Main />', () => {
  it('should render heading', () => {
    const { container } = renderWithTheme(<Main title="test" />);

    expect(screen.getByRole('heading', { name: /test/i })).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
