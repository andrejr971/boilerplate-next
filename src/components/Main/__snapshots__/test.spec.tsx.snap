// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render heading 1`] = `
.c0 {
  font-family: Saira,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
}

<h1
  class="c0"
>
  test
</h1>
`;
